README.md

```diff
@@ -1,247 +1,55 @@
-  def show
-    @post = Post.find(params[:id])
-
-    render json: @post
-  end
-end
-```
-
-In this case, Rails will look for a serializer named `PostSerializer`, and if
-it exists, use it to serialize the `Post`.
-
-### Specify a serializer
-
-If you wish to use a serializer other than the default, you can explicitly pass it to the renderer.
-
-#### 1. For a resource:
-
-```ruby
-  render json: @post, serializer: PostPreviewSerializer
-```
-
-#### 2. For an array resource:
-
-```ruby
-# Use the default `CollectionSerializer`, which will use `each_serializer` to
-# serialize each element
-render json: @posts, each_serializer: PostPreviewSerializer
-
-# Or, you can explicitly provide the collection serializer as well
-render json: @posts, serializer: CollectionSerializer, each_serializer: PostPreviewSerializer
-```
-
-### Meta
-
-If you want a `meta` attribute in your response, specify it in the `render`
-call:
-
-```ruby
-render json: @post, meta: { total: 10 }
-```
-
-The key can be customized using `meta_key` option.
-
-```ruby
-render json: @post, meta: { total: 10 }, meta_key: "custom_meta"
-```
-
-`meta` will only be included in your response if you are using an Adapter that supports `root`,
-as JsonAPI and Json adapters, the default adapter (Attributes) doesn't have `root`.
-
-### Using a serializer without `render`
-
-At times, you might want to use a serializer without rendering it to the view. For those cases, you can create an instance of `ActiveModel::SerializableResource` with
-the resource you want to be serialized and call `.serializable_hash`.
-
-```ruby
-def create
-  @message = current_user.messages.create!(message_params)
-  MessageCreationWorker.perform(serialized_message)
-  head 204
-end
-
-def serialized_message
-  ActiveModel::SerializableResource.new(@message).serializable_hash
-end
-```
-
-### Overriding association methods
-
-If you want to override any association, you can use:
-
-```ruby
-class PostSerializer < ActiveModel::Serializer
-  attributes :id, :body
-
-  has_many :comments
-
-  def comments
-    object.comments.active
-  end
-end
-```
-
-### Overriding attribute methods
-
-If you want to override any attribute, you can use:
-
-```ruby
-class PostSerializer < ActiveModel::Serializer
-  attributes :id, :body
-
-  has_many :comments
-
-  def body
-    object.body.downcase
-  end
-end
-```
-
-### Built in Adapters
-
-#### Attributes
+- [0.10 (master) Documentation](https://github.com/rails-api/active_model_serializers/tree/master)
+  - [![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/rails-api/active_model_serializers)
+  - [Guides](docs)
+- [0.9 (0-9-stable) Documentation](https://github.com/rails-api/active_model_serializers/tree/0-9-stable)
+- [0.8 (0-8-stable) Documentation](https://github.com/rails-api/active_model_serializers/tree/0-8-stable)
 
-It's the default adapter, it generates a json response without a root key.
-Doesn't follow any specific convention.
```

Comment:
re: https://github.com/rails-api/active_model_serializers/pull/1371#discussion_r47734219

It just said


```plain
#### Attributes

It's the default adapter, it generates a json response without a root key.
Doesn't follow any specific convention.
```

Thanks @duduribeiro